modules/xmpp/TraceablePeerConnection.js

```diff
@@ -322,11 +321,8 @@ TraceablePeerConnection.prototype.addStream = function (stream, ssrcInfo) {
     }
 };
 
-TraceablePeerConnection.prototype.removeStream = function (stream, stopStreams) {
+TraceablePeerConnection.prototype.removeStream = function (stream) {
     this.trace('removeStream', stream.id);
-    if (stopStreams) {
-        RTC.stopMediaStream(stream);
```

Comment:
Yeah, maybe it would be good if you could check if this is used currently. For me it looks like it's not and it could be removed.